### ISSUE TITLE
ANN: NetData v1.19.0-432 broke the Automatic Updater

### ISSUE THREAD

**Issue by prologic** [netdata/netdata#8056]
# If you are on NetData v1.19.0-432 read on...

#7939 broke the NetData Automatic Updater (_usually run via Cron_).

If you are on this version; you have two quick-fixes:

1. Amend the script with this patch:

```diff
diff --git a/packaging/installer/netdata-updater.sh b/packaging/installer/netdata-updater.sh
index fc85039d..0f01ada8 100755
--- a/packaging/installer/netdata-updater.sh
+++ b/packaging/installer/netdata-updater.sh
@@ -144,7 +144,7 @@ update() {
   info "Current Version: ${current_version}"
   info "Latest Version: ${latest_version}"

-  if [ "${latest_version}" -gt 0 ] && [ "${current_version}" -gt 0 ] && [ "${current_version}" -ge "${current_version}" ]; then
+  if [ "${latest_version}" -gt 0 ] && [ "${current_version}" -gt 0 ] && [ "${current_version}" -ge "${latest_version}" ]; then
     info "Newest version ${current_version} <= ${latest_version} is already installed"
   elif [ -n "${NETDATA_TARBALL_CHECKSUM}" ] && grep "${NETDATA_TARBALL_CHECKSUM}" sha256sum.txt >&3 2>&3; then
     info "Newest version is already installed"
```

This is usually located in `/usr/libexec/netdata/netdata-updater.sh` under a normal installation. You can easily find it by using `find`:

```sh
/netdata # find / -type f -name 'netdata-updater.sh'
/usr/libexec/netdata/netdata-updater.sh
```

2. Re-running the Installer which will also update your installation past `v1.19.0-432`

We apologise for any inconvenience caused.

----

Related Issues/Reports: #8027 #8029 #5053

**Comment by amoss** [comment]
Why is the test the opposite way around to the info message? 
* The test is current >= latest
* The info is current <= latest

**Comment by prologic** [comment]
> Why is the test the opposite way around to the info message?
> 
> * The test is current >= latest
> * The info is current <= latest

Fixed with #8078 8078
